automation_tools/__init__.py

```diff
@@ -3115,13 +3116,39 @@ def download_manifest(url=None, consumer=None):
                          ' session and distributor hash')
 
 
+def assert_downloading_manifest():
+    """Makes sure that downloaded manifest matches manifest name from a reference file."""
```

Comment:
This is not really true statement, as we don't check a downloaded manifest - this method checks that the portal account has the required subscriptions attached